Extensions/Physics2Behavior/physics2runtimebehavior.ts

```diff
@@ -393,7 +398,12 @@ namespace gdjs {
       }
     }
 
+    onActivate() {
+      this._updateBodyFromObject();
+    }
+
     onDestroy() {
+      this.destroyedDuringFrameLogic = true;
       this.onDeActivate();
     }
```

Comment:
Should `activate(enable: boolean): void` be called instead?
It will change the value returned by `activated(): boolean` and make `destroyedDuringFrameLogic` useless. It will ensure that `onDeActivate()` won't be called twice.

Reply:
I would avoid to call `activate(false)` because:
- this is an operation that could involves more stuff later. This is implemented in the runtime behavior, and here we call onDeActivate because "it's the same code", but not because we want to deactivate the behavior. In other words, the code paths could differ in the future, the fact that they are the same is just a coincidence.
- onDestroy should already be called only once anyway?

Reply:
Ok, deleting a deactivated instance seem to be a good test case.